src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[new Random().nextInt(10)];
```

Comment:
Same, let's make constant for 10

Reply:
done

Reply:
to create constant: static final int RANGE = 10 I couldn't. 
If I create final int RANGE = 10 -> mvn package gives an error message. 
I did final int Max = 10; but all constant's name must be UpperCase. 
I'm like in a vicious circle...

Reply:
done